src/main/java/staffconnect/model/ModelManager.java

```diff
@@ -119,13 +123,24 @@ public void setPerson(Person target, Person editedPerson) {
      */
     @Override
     public ObservableList<Person> getFilteredPersonList() {
-        return filteredPersons;
+        return sortedFilteredPersons;
     }
```

Comment:
is it possible to have another getter for sorted list? e.g `getSortedPersonList()`

Reply:
:troll_face: